enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,231 @@
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This document describes how the [IBM Cloud][ibm-cloud-website] can be leveraged to provide infrastructure to deploy
+OpenShift on the [Power Virtual Server][power-vs-website] offering. Power Virtual Server, or Power VS, is an IBM Cloud
+offering that aims to provide customers interested in running secure and flexible workloads on IBM Power (ppc64le)
+hardware without having to develop and maintain on premise datacenters. The goal of this enhancement would be to
+leverage the IBM Cloud APIs to deploy an OpenShift cluster on this cloud instance hardware. In addition, the APIs would
+be leveraged for cluster resizing operations post installation.
+
+## Motivation
+
+- Current deployments of OpenShift on Power use UPI, with the exception of the development only configuration of IPI on
+libvirt.
+- Currently, the Multi-Arch team is stuck using a development configuration (IPI on libvirt) to cover our CI workloads
+for OpenShift on ppc64le verification. This would help diversify our CI effort.
+- A non-x86 IPI installer would serve as a first step towards improving the ease of deployments on non-x86 hadware.
+- Power hardware is especially targeted to AI and secure workloads, so building out cloud capabilities will help expand
+our workload footprint.
+- IPI is generally easier to install and set up quickly for demos and proof of concepts, when can help lower the barrier
+of entry for power optimized workloads.
+
+### Goals
+
+- Provide a way to install OpenShift on Power VS infrastructure using the OpenShift installer with IPI provisioned
+against IBM Cloud APIs.
+- Implement the cluster-api provider to provide scaling and managing the cluster nodes post install.
+
+### Non-Goals
+
+- UPI support. We've already documented how to install on pp64le hardware using UPI.
+
+## Proposal
+
+In order to leverage the Power VS platform, we will first need to implement an IBM Cloud Provider. This provider will
+interface with IBM Cloud APIs and ultimately enable the OpenShift Installer to provision VM resources using Power VS,
+that will be used as worker and masters of the cluster. Unlike some of the existing providers, this depends on image
+availability of CoreOS in the PowerVS cloud the OVA format. There are currently 2 proposals of how that can be
+achieved.
+
+The expected workflow would be as follows:
+1. The installer is invoked with credentials for ones IBM Cloud environment.
+2. Once invoked, the installer interacts with a terraform provider to create a new Power VS Service Instance.
+3. The installer will then import a CoreOS OVA boot image from an existing cloud store to the service instance.
```

Comment:
do we want to break this down into the detail that the ova would be first copied from the cloud-object-storage to the PowerVS service instance, a bootimage would be created and then the VM  would be provisioned ?